### PR TITLE
Bf-025 Add Seen resource

### PR DESCRIPTION
**Pull request by shockolegario**

### CODE REVIEW

**Comment by shockolegario** [comment]
for this task I use the ngResource

File: blimflix/client/app/providers/seen.factory.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+   * Avoid anonymous functions as callbacks
+   *   https://github.com/johnpapa/angular-styleguide#style-y024
+   */
+  /* @ngInject */
+  function seen($resource) {
+
+    return $resource('http://localhost:3333/users/:id/seen/:idM',{id:'@id', idM:'@idM'}, {
+      'query': { method: 'GET', isArray: true},
+      'save': { method: 'POST' },
+      'remove': { method:'DELETE'}
```
Comment (devRodSoft) [inline blimflix/client/app/providers/seen.factory.js]: Just a suggestion: When you have variable/object assign value is a good practice code like this example, look the right values are align with the same spaces after the ":", this way is more readily :) and check the space after/before the curly braces {}
```
'key': { method: 'GET', isArray: true },
'key': { method: 'GET', isArray: true },
'key': { method: 'GET', isArray: true }
```

**Review by CarlosPadilla — APPROVED** [review]

**Review by devRodSoft — CHANGES_REQUESTED** [review]

File: blimflix/client/app/providers/seen.factory.js (hunk 1 above)
Comment (shockolegario) [reply blimflix/client/app/providers/seen.factory.js]: ready

**Comment by shockolegario** [comment]
Ready, I corrected the observation

**Review by shockolegario — COMMENTED** [review]
ready

**Review by lordpixel — APPROVED** [review]

**Review by devRodSoft — APPROVED** [review]

**Review by devRodSoft — APPROVED** [review]

**Pull request merged**